src/index.ts

```diff
@@ -204,11 +205,34 @@ export type {
 export type { AutoEncrypter } from './client-side-encryption/auto_encrypter';
 export type { AutoEncryptionOptions } from './client-side-encryption/auto_encrypter';
 export type { AutoEncryptionExtraOptions } from './client-side-encryption/auto_encrypter';
+export type {
+  AWSEncryptionKeyOptions,
+  AzureEncryptionKeyOptions,
+  ClientEncryptionCreateDataKeyProviderOptions,
+  ClientEncryptionEncryptOptions,
+  ClientEncryptionOptions,
+  ClientEncryptionRewrapManyDataKeyProviderOptions,
+  ClientEncryptionRewrapManyDataKeyResult,
+  DataKey,
+  GCPEncryptionKeyOptions,
+  RangeOptions
+} from './client-side-encryption/client_encryption';
```

Comment:
AC has these as needing to be exported:

```
interface ClientEncryptionCreateDataKeyCallback
interface ClientEncryptionEncryptCallback
interface ClientEncryptionDecryptCallback
```

I'm assuming that's wrong in the ticket as well since those methods are becoming async/await?

Reply:
I updated the ticket description to be more clear that we weren't exporting those types.